### ISSUE TITLE
Jekyll wraps content in <p> tags inconsistently.

### ISSUE THREAD

**Issue by itsmaxymoo** [jekyll/jekyll#7714]
# Environment:
| Software | Versions |
| --- | --- |
| Operating System | Ubuntu 18.04.2 LTS with Linux 4.15.0-51-generic |
| Jekyll | jekyll 3.8.5 |

# Expected Behavior
Jekyll wraps `(collection item).content` in `<p>` tags consistently.

# Current Behavior
Including `{{ (collection item).content }}` in a normal page or in the content of
another collection file renders the original collection item's content wrapped in `<p>` tags.
Only in a post file however, the original collection item's content is **not** wrapped in `<p>` tags.

In short, `{{ (collection item).content }}` becomes:

| On a page/other collection | In a post |
| --- | --- |
| `<p>Some content</p>` | `Some content` |

This leads to inconsistent styling.
